model-server/src/main/kotlin/org/modelix/model/server/handlers/RepositoriesManager.kt

```diff
@@ -56,6 +56,10 @@ class RepositoriesManager(val client: LocalModelClient) {
     private val objectStore: IDeserializingKeyValueStore get() = client.storeCache
     val inMemoryModels = InMemoryModels()
 
+    fun dispose() {
```

Comment:
This dispose is never called. Do you intend it to call in tests or Main.kt on shutdown?

Reply:
The method should be called when the RepositoriesManager instance is not used anymore. Tests would be one place. We have to search for all places where an instance is created and add a `dispose()` call.

Reply:
Is the idea to do this in this PR or later?

Reply:
There are no cases where it's really necessary. I added a TODO in the code. It's save to ignore for now. Not even worth a ticket.